### ISSUE TITLE
Allow specification of censoring model when calculating IPCW weighted Brier/Graf Score

### ISSUE THREAD

**Issue by adibender** [mlr-org/mlr3proba#164]
In theory, it would also be possible to specify the learner which is used to learn the censoring model, tune the parameters, etc.... For now I'd restrict to simple parametric learner, e.g. `coxph`, but raises questions. E.g. what to do in n <p cases.

**Comment by RaphaelS1** [comment]
Do we really want to do this? I know this is possible in {pec} but it has no good theoretical justification. How do you validate the censoring model? What about data bias which just gets propagated forward by the second learner. I'm not convinced this is something we should implement

**Comment by adibender** [comment]
Yes, vlaidation of the censoring model is a problem. But we should allow it should be able to 
a) recreate results from literature
b) use it for comparison when they develop alternatives or similar 

Is it hard to do technically?

**Comment by RaphaelS1** [comment]
> Is it hard to do technically?

Nope

>  recreate results from literature

It's an interesting argument. Okay, let's do it. I'll bump this up my list

**Comment by bblodfon** [comment]
Moved to `mlr3survival`
